specs/capella/beacon-chain.md

```diff
@@ -154,7 +154,9 @@ class ExecutionPayload(Container):
     # Extra payload fields
     block_hash: Hash32  # Hash of execution block
     transactions: List[Transaction, MAX_TRANSACTIONS_PER_PAYLOAD]
+    transactions_hash: Bytes32  # [New in Capella]
     withdrawals: List[Withdrawal, MAX_WITHDRAWALS_PER_PAYLOAD]  # [New in Capella]
+    withdrawals_hash: Bytes32  # [New in Capella]
 ```
```

Comment:
we'd also need to extend the engine API to include validation of these values

Reply:
Yes, engine API is being extended as part of Capella anyway, so that shouldn't be a problem.

Reply:
Matching engine API PR: https://github.com/ethereum/execution-apis/pull/318